Documentation Update

mryindles opened the pull request:
Rolled out big changes to our user stories document to reflect the drastic change in direction the project has taken. Created readme.md as per project instructions, and moved original open tournament developer readme to our documentation folder. Only making this a pull request because this needs to get rolled out to master, and master is a protected branch. Please view all documentation changes, and let me know you approve of them.

mryindles committed:
Documentation Update

Rolled out big changes to our user stories document to reflect the drastic change in direction the project has taken. Created readme.md as per project instructions, and moved original open tournament developer readme to our documentation folder.

MMercer8989 reviewed (approved):
For the most part, everything looks and reads adequately, though I would suggest capitalizing The header for the test coverage section and the project info section to match the formatting for the rest of the document.

MMercer8989 commented:
Additionally, the updates to the user stories document seem alright. I might suggest taking out the word incentivizing for C4P-44, and changing instances of "Matthew" to "Matt"(my full name is Matt) to avoid any confusion when grading. other than those two nitpicks though, I would say it's good to go.

daniellcurto reviewed (changes requested):
I do agree we did not realize what we were doing wrong until the professor said something. At least we realized it now and not after Dr. Buck graded our project. So far,  I am not sure what else to say about this commit.

mryindles committed:
Pull Request Update

Made slight changes, and edits to these documents based on feedback received during pull request.
